docs/user/explanations/data-model.rst

```diff
@@ -483,11 +483,10 @@ Typical example:
 
    # 'Stream Resource' document
    {'data_key': 'detector_1',
-    'spec': 'AD_HDF5',
+    'mimetype': 'application/x-hdf5',
     'root': '/GPFS/DATA/Andor/',
```

Comment:
I think `root` and `resource_path` should be refactored into a `uri`.

Reply:
ah, missed this bit. thanks!